plugins/social.server.ts

```diff
@@ -0,0 +1,16 @@
+import { sendRedirect } from 'h3'
+
+export default defineNuxtPlugin(async (nuxtApp) => {
+  const route = useRoute()
+  if (!route.params.server)
+    return
+
+  const req = nuxtApp.ssrContext!.event.node.req
```

Comment:
Why not using useRequestEvent() here @danielroe ?
https://nuxt.com/docs/api/composables/use-request-event

Reply:
Or even https://nuxt.com/docs/api/composables/use-request-headers ?

Reply:
much better!